AENS.md

```diff
@@ -347,6 +376,30 @@ the name enters the `revoked` state. After a fixed timeout of
 2016 blocks, the name will be available for claiming again.
 
 
+### Subname
+
+```
+ ------------ ----
+| name       | 63 |
+ ------------ ----
+| definition |    |
+ ------------ ----
+```
+
+The `subname` transaction MUST be signed by the owner of the `name` entry.
+
+`subname` transaction is used to define a tree of subnames below `name` entry
```

Comment:
If I read this correctly, "tree of subnames" means no subname without a parent. Ok.

Reply:
See also https://github.com/aeternity/protocol/pull/392#discussion_r311969444